.goreleaser.yml

```diff
@@ -34,8 +34,7 @@ changelog:
       - Merge branch
 dockers:
 - image_templates:
-  - 'skibish/ddns:{{ .Tag }}'
-  - 'skibish/ddns:latest'
```

Comment:
I would propose not to remove the `latest` tag because this might break the current users who already pull by `latest` tag

Reply:
okay, sounds reasonable